packages/expo-age-range/android/src/main/java/expo/modules/agerange/AgeRangeModule.kt

```diff
@@ -0,0 +1,63 @@
+package expo.modules.agerange
+
+import android.content.Context
+import com.google.android.play.agesignals.AgeSignalsException
+import com.google.android.play.agesignals.AgeSignalsManagerFactory
+import com.google.android.play.agesignals.AgeSignalsRequest
+import expo.modules.kotlin.Promise
+import expo.modules.kotlin.exception.Exceptions
+import expo.modules.kotlin.exception.toCodedException
+import expo.modules.kotlin.modules.Module
+import expo.modules.kotlin.modules.ModuleDefinition
+import expo.modules.kotlin.exception.CodedException
+
+class AgeRangeModule : Module() {
+  private val context: Context
+    get() = appContext.reactContext ?: throw Exceptions.ReactContextLost()
+
+  override fun definition() = ModuleDefinition {
+    Name("ExpoAgeRange")
+
+    AsyncFunction("requestAgeRangeAsync") { _: Any, promise: Promise ->
+      val ageSignalsManager =
+        AgeSignalsManagerFactory.create(context.applicationContext)
+
+      requestAgeRange(
+        ageSignalsManager = ageSignalsManager,
+        onSuccess = { result -> promise.resolve(result) },
+        onError = { exception -> promise.reject(exception) },
+        onCancelled = { promise.reject(AgeRangeTaskCancelledException()) }
+      )
+    }
+  }
+}
+
+fun requestAgeRange(
+  ageSignalsManager: com.google.android.play.agesignals.AgeSignalsManager,
```

Comment:
```suggestion
  ageSignalsManager: AgeSignalsManager,
```
Let's import it and remove the fully qualified name